pkg/build/api/validation/validation.go

```diff
@@ -500,11 +500,14 @@ func validateTrigger(trigger *buildapi.BuildTriggerPolicy, buildFrom *kapi.Objec
 	return allErrs
 }
 
-func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path) field.ErrorList {
+func validateWebHook(webHook *buildapi.WebHookTrigger, fldPath *field.Path, allowEnv bool) field.ErrorList {
```

Comment:
On the describer, for both the github and generic webhook, there is nothing done for them in general, including the existing secret field:

https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/describer.go#L387-L388

@bparees @smarterclayton :
Is the addition of allowEnv enough to warrant a change here (or should something have been done for these even prior to the addition of allowEnv) ?